md/deploy-bundle.md

```diff
@@ -23,10 +23,89 @@ It is useful to update the configuration, locally or from a remote computer.
+    - url: the URL, including the RDBMS server address, RDBMS server port and database (schema) name.
+5. Repeat operations 2. to 4. for business data data source
+
+JTA data source (managed by Bitronix)
+1. Open the file TOMCAT_HOME/conf/bitronix-resources.properties
+2. Remove or comment out the lines regarding the h2 database.
+3. Uncomment the line matching your RDBMS.
+4. Update the value for each of the following settings:
+    - resource.ds1.driverProperties.user: your RDBMS user name.
+    - resource.ds1.driverProperties.password: your RDBMS password.
+    - resource.ds1.driverProperties.serverName: the address (IP or hostname) of your RDBMS	server.
+    - resource.ds1.driverProperties.portNumber: put the port of your RDBMS server.
+    - resource.ds1.driverProperties.databaseName: put the database name.
+5. Repeat operations 2. to 4. for business data data source (resources.ds2)
+
+### Configure RDBMS vendor
+
+1. Open the file TOMCAT_HOME/bin/setenv.(bat|sh)
+2. Change the value of sysprop.bonita.db.vendor according to your RDBMS vendor
+3. Change the value of sysprop.bonita.bdm.db.vendor according to your RDBMS vendor
+
+### Add Jdbc driver (only for Microsoft SQL Server or Oracle)
+If and only if you are using Microsoft SQL Server or Oracle, you need to add your jdbc driver in
+TOMCAT_HOME/server/lib. Other RDBMS vendor drivers are already included in bonita.war
+
+<a id="wildfly-installation" />
+
+## Wildfly installation
+
+### Install Bonita BPM Platform in Wildfly
+Copy all files and directories from DEPLOY_ZIP_HOME/wildfly-10.1.0.Final to your Wildfly root directory (WILDFLY_HOME).
+
+### Configure data sources
+1. Open the file WILDFLY_HOME/standalone/configuration/standalone.xml
+2. Remove or comment out the default definition for h2.
+3. Uncomment the settings matching your RDBMS vendor.
+4. Modify the values for following settings to your configuration: server address, server port, database name, user name and password.
+5. Repeat operations 2. to 4. for business data data source
+
+### Configure RDBMS vendor
+Open WILDFLY_HOME/standalone/configuration/standalone.xml and set the value of sysprop.bonita.db.vendor and 
+sysprop.bonita.bdm.db.vendor according to your RDBMS vendor (look for <system-properties> tag).
+
+### Add Jdbc driver
+1. Create a folder structure under WILDFLY_HOME/modules folder. Refer to the table below to identify the folders to create. 
+The last folder is named `main` for all JDBC drivers.
+| Database vendor | Module folders              | 	Module description file
+| PostgreSQL	  | modules/org/postgresql/main | [module.xml](images/special_code/postgresql/module.xml)
+| Oracle	      | modules/com/oracle/main     | [module.xml](images/special_code/oracle/module.xml)
+| SQL Server	  | modules/com/sqlserver/main  | [module.xml](images/special_code/sqlserver/module.xml)
+| MySQL	          | modules/com/mysql/main      | [module.xml](images/special_code/mysql/module.xml)
+
```

Comment:
This table does not display well: http://doc.rd.lan:50280/?page=deploy-bundle :disappointed: